src/solution/react-dom/react-dom/README.md

```diff
@@ -0,0 +1,10 @@
+# ReactDOM
+
+Here you will find the proposed solution to the implementation of ReactDOM without a class-cache.
+
+This version of ReactDOM is implemented without a class-cache. The implementation is a little bit more advanced than 
+the class-cache version. Stateful components are stored as instances on the `VCompositeNode` instead of a 
+"centralized" cache, as we did with the class-cache. We there need to traverse the tree of virtual-DOM nodes and 
```

Comment:
«We there need to..». Mente du «Therefore»? @eirikv

Reply:
"We there, who, what, were, when..." 😏 Jupp, mente nok noe i den duren... 😉